### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Bassoonian**
Removes Mulches, Dynamax Candy and Mints from the "Existing item data but missing effects" category in upcoming's README, as said features do in fact have functionality in upcoming.

## **Discord contact info**
bassoonian

### CODE REVIEW

**Comment by AsparagusEduardo** [comment]
Can you add merrp's Guillotine to the feature branch list?

**Comment by Bassoonian** [comment]
Oh, I thought it was made specifically for expansion. Does it exist as a standalone branch, too?

**Comment by AsparagusEduardo** [comment]
> Oh, I thought it was made specifically for expansion. Does it exist as a standalone branch, too?

Yep
https://github.com/aarant/pokeemerald/tree/guillotine

**Comment by Bassoonian** [comment]
Done!

File: README.md [hunk 1]
```diff
@@ -141,6 +138,7 @@ Based off RHH's pokeemerald-expansion v1.7.3 https://github.com/rh-hideout/pokee
             - *Dark Mode*.
     - [Nature Colors](https://github.com/DizzyEggg/pokeemerald/tree/nature_color) in summary screen by @DizzyEggg
     - [Dynamic Multichoice](https://github.com/SBird1337/pokeemerald/tree/feature/dynmulti) by @SBird1337
+    - [Guillotine](https://github.com/aarant/pokeemerald/tree/guillotine) by @aarant
```
Comment (AsparagusEduardo) [inline README.md:141]:
```suggestion
    - [Guillotine](https://github.com/aarant/pokeemerald/tree/guillotine) (Decapitalization) by @aarant
```

**Review by AsparagusEduardo — APPROVED** [review]

**Pull request merged**